config/routes.rb

```diff
@@ -35,7 +35,9 @@
   post "/orders", to: "orders#create"
   get "/orders/:id", to: "orders#show"
 
-  get "/users/new", to: "users#new"
-  post "/users/new", to: "users#create"
+  get "login", to: "sessions#new"
+
+  get "/register", to: "users#new"
+  post "/register", to: "users#create"
```

Comment:
The only problem I see with this is that by changing the path to '/register' we lose the ability to refactor this using `resources: :users`